src/jlapi.c

```diff
@@ -680,6 +684,11 @@ static void rr_detach_teleport(void) {
 
 JL_DLLEXPORT int jl_repl_entrypoint(int argc, char *argv[])
 {
+#ifdef USE_TRACY
```

Comment:
I think it would be very useful to have an environment variable that allows changing `jl_timing_enable_mask` before we're running Julia user code.  Something like `JULIA_TIMING_SUBSYSTEMS=INFERENCE,GC,METHOD_MATCH,-SYSIMG_LOAD` which then gets decoded into turning on the first three elements and turning the last one off (relative to the default).  That way, we can have sensible defaults for `jl_timing_enable_mask` without limiting flexibility.

Reply:
A great idea! Let me see what I can whip up.